ices/80664.rs

```diff
@@ -0,0 +1,7 @@
+mod inner {
+    pub struct Public;
+}
+
+pub use inner::Public as Reexported;
+
+fn main() {}
```

Comment:
This one needs to be passed into rustdoc with `--output-format json` in order to ICE, you use a `.sh` file to do this, e.g. https://github.com/rust-lang/glacier/blob/master/ices/52334.sh

Reply:
Done